CONTRIBUTING.md

```diff
@@ -120,11 +126,12 @@ score.
 - Document your reasoning behind the changes. Explain why you wrote the code in
   the way you did. The code should explain what it does.
 - If there's an existing issue related to the pull request, reference to it by
-  adding something like `References/Closes/Fixes/Resolves #305`, where 305 is the issue number.
+  adding something like `References/Closes/Fixes/Resolves #305`, where 305 is
+  the issue number.
   [More info here](https://github.com/blog/1506-closing-issues-via-pull-requests).
 - Please fill out the PR Template when making a PR.
 - All commits in a pull request will be squashed when merged, but when your PR
-  is approved and passes our CI, it will be live on production!
+  is approved and passes our CI, it will eventually be live on production!
```

Comment:
as PRs merged don't always correspond to deploys, I felt it best to add `eventually`